extension/user/getUser.js

```diff
@@ -0,0 +1,23 @@
+const {EACCESS, ENOTFOUND, customError} = require('../error')
+/**
+ * @param {SDKContext} context
+ * @param {Object} input
+ * @param {function} cb
+ */
+module.exports = (context, input, cb) => {
+  if (!context.meta.userId) {
+    return cb(customError(EACCESS, 'Unauthorized access'))
+  }
+
+  context.storage.extension.get(context.meta.userId, (err, user) => {
+    if (err) {
+      return cb(err)
```

Comment:
Should not expose internal error messages, but instead log them and respond with an EINTERNAL error.

Reply:
done